src/ontology/mondo-ingest.Makefile

```diff
@@ -476,6 +487,7 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_map
 	--mondo-terms-path $(REPORTDIR)/mirror_signature-mondo.tsv \
 	--slurp-dir-path slurp/ \
 	--outpath $@
+.PRECIOUS: slurp/%.tsv
```

Comment:
## Bug fix: `slurp/icd11foundation.tsv`
- [ ] Review @matentzn @twhetzel 

@matentzn Saw that I'd observed `rm slurp/icd11foundation.tsv` after doing `slurp-icd11foundation` and recommended that I add a `.PRECIOUS` marker on the goal. That fixed the issue.

A few other things:
1. **Why just ICD11?**: I don't understand why this was a problem only for `icd11foundation`.
2. **Why did this work?**: After doing some minor reading, I'm not 100% sure I understand why `.PRECIOUS` was needed at all. Since our goal is `slurp/%.tsv`, it would seem to me that `slurp/icd11foundation.tsv` wouldn't be considered an intermediate. [ChatGPT thinks](https://chat.openai.com/share/d21169e2-3d52-4a3c-b6b6-9a1aafd58e0f) that it's considered an intermediate because (1) it has a `%` in it, and (2) because it's being called by a `.PHONY` goal. But I'm still not convinced. We have many such cases of (1) and (2) throughout our pipeline, but I do not see things getting `rm`'ed afterwards.
3. **`rm` issue**: I checked the build log from last week, and at the end it showed `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv imports/ro_terms_combined.txt `. After adding that 1 `.PRECIOUS`, now the slurp file is no longer being removed, but the other two still are: `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv imports/ro_terms_combined.txt`. I think we want these, so I'm going to apply the same fix for both of those as well. I pasted more of the log from `build-mondo-ingest` below just in case it's of any value.

<details><summary>Log related to (3)</summary>
<p>

I have the full logs saved as `.txt` files also if interested

From the build last week:
```
Release files are now in ../.. - now you should commit, push and make a release         on your git hosting site such as GitHub or GitLab
rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
Mondo Ingest has been fully completed
```

From the build today:
```
Release files are now in ../.. - now you should commit, push and make a release         on your git hosting site such as GitHub or GitLab
rm reports/icd11foundation_exclusion_reasons.robot.template.tsv imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
Mondo Ingest has been fully completed
```

</p>
</details>